src/Compilers/Core/Portable/Emit/EmitOptions.cs

```diff
@@ -89,6 +91,35 @@ public sealed class EmitOptions : IEquatable<EmitOptions>
         /// </summary>
         public string RuntimeMetadataVersion { get; private set; }
 
+        // 1.2 BACKCOMPAT OVERLOAD -- DO NOT TOUCH
```

Comment:
Isn't the backcompat already handled by the fact that the new `string instrument` parameter to the constructor is optional?